client/src/components/postcall/PostCallOneQuestionPoll.tsx

```diff
@@ -22,7 +22,7 @@ export const PostCallOneQuestionPoll: React.FunctionComponent<PostCallOneQuestio
 
   const submitSurveyResponse = async (): Promise<void> => {
     await submitSurveyResponseUtil(props.acsUserId, pollResponse, props.callId);
-    window.location.replace('/book');
+    // window.location.replace('/book');
```

Comment:
Why are we commenting this?

Reply:
Sorry that was by mistake.  will revert back